File: <100/002/002.py
def main() -> None:
    upper_limit: int = 4000000
    answer: int = 0
    Fib_2: int = 1
    Fib_1: int = 2
    while True:
        Fib: int = Fib_1 + Fib_2
        if Fib > upper_limit: break
        if (Fib&1) == 0: answer += Fib
        Fib_2, Fib_1 = Fib_1, Fib

    print(answer)

    return


if __name__ == '__main__':
    main()

File: <100/002/002.jl
function main():: Nothing
  Fib_2:: Int = 1
  Fib_1:: Int = 2
  Fib::Int = 0
  answer:: Int = Fib_1
  upper_limit:: Int = 4000000

  while true
    Fib = Fib_1 + Fib_2
    if Fib > upper_limit
      break
    end
    if Fib |> iseven
      answer += Fib
    end
    Fib_2, Fib_1 = Fib_1, Fib
  end
  
  answer |> println
  return nothing
end

@time main()
